config/locales/en.yml

```diff
@@ -557,6 +563,8 @@ en:
         date_from: Date from
         date_to: Date to
         date_requested: Date requested
+      data_request_area:
+        data_request_area_type: Where is the data you are requesting from?
```

Comment:
This doesn't seem to be quite the right question when it's not about the explicit location.